homeassistant/components/bluetooth_tracker/manifest.json

```diff
@@ -5,5 +5,6 @@
   "documentation": "https://www.home-assistant.io/integrations/bluetooth_tracker",
   "iot_class": "local_polling",
   "loggers": ["bluetooth", "bt_proximity"],
-  "requirements": ["bt-proximity==0.2.1", "PyBluez==0.22"]
+  "requirements": ["bt-proximity==0.2.1", "git+https://github.com/pybluez/pybluez.git@master#pybluez"],
+  "version": "1.0.0"
```

Comment:
We can't accept requirements that aren't packaged releases.

I suggest splitting this PR so the device tracker change can be merged

Reply:
Yeah I expected that and removed that one commit.

Do you know if, besides of HACS or waiting and praying for the owner of PyBluez releasing another official version, there is anything that can be done?

Reply:
Fork PyBluez and create a new GitHub org to maintain it

Reply:
If you are interested in maintaining it, I would try to reach out to the existing repo owners first

Reply:
well ok. I'm not really interested in maintaining it (there are already new maintainers), mostly because its not my main language/environment. Just saw (like some others), that it got updated since the last release of 0.23 in 2019 and is even compatible to Python 3.11. sadly the PyPi release is kinda stale since 2021, because the new maintainers on GitHub dont have access to the PyPi account for example.

Reply:
It looks like its a fragmented mess https://pypi.org/search/?q=pybluez with no fork clearly alive.